src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/TagHelperConstants.cs

```diff
@@ -33,5 +38,8 @@ public static class TagHelperConstants
         public const string NhsValidationSummaryTitle = "nhsuk-error-summary__title";
         public const string NhsValidationSummaryList = "nhsuk-error-summary__list";
         public const string NhsValidationInputError = "nhsuk-input--error";
+        public const string NhsBackLinkLink = "nhsuk-back-link__link";
+        public const string NhsIcon = "nhsuk-icon";
+        public const string NhsIconChevronLeft = "nhsuk-icon__chevron-left";
     }
 }
```

Comment:
I understand making your constants `public` to be consistent with the existing ones but I'm not sure why all the existing constants are `public` when they could be `internal`. Personally, I'd probably make them all `internal` but I'll leave that up to you.

Reply:
Thats fine with me, changing them to internal 👍